feature/google/src/main/java/com/revenuecat/purchases/google/BillingWrapper.kt

```diff
@@ -157,8 +157,8 @@ class BillingWrapper(
         log(LogIntent.DEBUG, OfferingStrings.FETCHING_PRODUCTS.format(productIds.joinToString()))
         executeRequestOnUIThread { connectionError ->
             if (connectionError == null) {
-                val googleType = productType.toGoogleProductType() ?: BillingClient.ProductType.SUBS
-                val params = googleType.buildQueryProductDetailsParams(productIds)
+                val googleType = productType.toGoogleProductType() ?: BillingClient.ProductType.INAPP
+                val params = googleType.buildQueryProductDetailsParams(nonEmptyProductIds)
```

Comment:
This was also making a test fail

Reply:
Hmm we are exiting early if all product ids were empty. I guess it failed if some of the product ids were empty?